src/CMakeLists.txt

```diff
@@ -64,10 +64,6 @@ endif()
 target_link_libraries(qtadvanceddocking PUBLIC Qt${QT_VERSION_MAJOR}::Core 
                                                Qt${QT_VERSION_MAJOR}::Gui 
                                                Qt${QT_VERSION_MAJOR}::Widgets)
-if(UNIX AND NOT APPLE)
-    target_link_libraries(qtadvanceddocking PUBLIC Qt${QT_VERSION_MAJOR}::X11Extras)
-	target_link_libraries(qtadvanceddocking PRIVATE xcb)
```

Comment:
This doesn't seem to be explicitly needed, as it worked in my tests for 5.15.4, but not sure if that is the case for pre-5.15 releases.